proto/frequenz/api/common/v1/metrics/metric_sample.proto

```diff
@@ -63,6 +63,19 @@ message MetricValueVariant {
 }
 
 // List of supported metrics.
+//
+// AC energy metrics information:
+//
+// * This energy metric is reported directly from the component, and not a
+//  result of aggregations in our systems. If a component does not have this
+//  metric, this field cannot be populated.
+//
+//  * Components that provide energy metrics reset this metric from time to
+//  time. This behaviour is specific to each component model. E.g., some
+//  components reset it on UTC 00:00:00.
+//
+//  * This energy metric does not specify the timestamp since when the energy
+//  was being accumulated, and therefore can be inconsistent.
```

Comment:
LGTM in general. Should we make it more visible by making it a [note](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN#notes_warnings_and_callouts)?

That way it looks more like a notice, and not general documentation about this enum.

Reply:
A bunch of CI test are failing now that I was trying to mark it down as a note. Should I change it back?